### DISCUSSION TITLE
Possible to assign multiple tenants to Site?

### DISCUSSION THREAD

**Discussion by RyanMountain** [netbox-community/netbox#10159] [Q&A]
Hi,

Is it possible to assign multiple tenants to sites?

Reason for this is say I have a site where multiple tenants are sharing so this leads to them also sharing the equipment, circuits and other assets etc.

A model I have seen advises not to assign a tenant to site in this occasion and assign the devices to the site. But then how do the devices get reported to the tenants so we can cost them x amount for sharing equipment and the stat for 'Site' will not appear for the tenant in as this tenant will not have been assinged to the site?

Am I going about this the wrong way or missing something here?

Thanks in advance, Ryan.

**Comment by cyberndj** [comment]
Hi Ryan,

Might I suggest viewing Sites as a physical location (ie building with a unique address) and Locations as "areas" within a Site. You can create a Location for each tenant and assign each one of those Locations to the site. Then you could do nested locations to better represent that single building's floors/sections/rooms/areas/etc. in a method that suits your needs.

Thanks!
-Jake
